account_payment_notification/data/mail_template.xml

```diff
@@ -19,6 +19,9 @@
             </p>
             <p t-else="">
                 Your payment order was processed and sent to the bank.
+
```

Comment:
```suggestion
<br/>
```

Now looks like this:
![image](https://github.com/user-attachments/assets/5b60d1c8-1c53-4df5-a9bb-20b9b59381ca)